contracts/WakuRln.sol

```diff
@@ -7,25 +7,46 @@ import {Ownable} from "openzeppelin-contracts/contracts/access/Ownable.sol";
 
 error NotImplemented();
 
+error InvalidIdCommitmentIndex(uint256 idCommitment, uint256 index);
+
+error MembershipStillActive(uint256 idCommitment);
+
 contract WakuRln is Ownable, RlnBase {
     uint16 public immutable contractIndex;
 
+    /// @notice The default TTL period in seconds for a membership
+    uint256 public immutable MEMBERSHIP_TTL;
```

Comment:
need to think about this, does it need to be `uint256`?
`uint40` works fine for timestamps, and especially because `contractIndex` is a uint16, they both will be packed into the same storage slot.

Reply:
Thanks for the review, I still haven't focused on the variable types since I wanted to have the basic flow set first!
Will take into account these comments in next commits!